Add `Expert Global Taiko Showdown 2025 Concludes` news post (#13208)
ppy/osu-wiki: news/2025/2025-05-03-expert-global-taiko-showdown-2025-concludes.md

- I played against [chibamasu](https://osu.ppy.sh/users/16067522) in Round of 32, and I was impressed when they FC'd every FM map with HDHR and when they SS'd the NM5 map. Their HD skill and overall consistency really surprised me. Their HD2 play at the All-Star Showcase was impressive as well.

</div>

![](/wiki/shared/news/2025-05-03-expert-global-taiko-showdown-2025-concludes/interview_4sbet1.png)

[4sbet1](https://osu.ppy.sh/users/11563671) was one of the head mappoolers for this year's EGTS, having helped tremendously in terms of improving communication with Japanese users as well as having worked upon a huge part of the general mappooling work with [ekumea1123](https://osu.ppy.sh/users/9119501), especially for later stages.

<div class="news-chat-quote__text-container">

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6177263" style="background-image: url('/wiki/shared/avatars/Kasumi-sama.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/6177263">Kasumi-sama</a></p>

> Despite being a top player for years, this mappooling experience has been your first ever on an international tournament. Coming in with such a big amount of restrictions, with the mappool being fully custom-mapped, involving a lot of original songs to balance into it as well, how did it feel organis   ing all of this?

Reviewer: ```suggestion
Despite being a top player for years, this mappooling experience has been your first ever on an international tournament. Coming in with such a big amount of restrictions, with the mappool being fully custom-mapped, involving a lot of original songs to balance into it as well, how did it feel organising all of this?
```

Reviewer: i hope this is an accident

Author: wth

Despite some narrow misses at the start and at the end of HR2, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) didn't drop a single time, reducing his miss count from 73 to 7 and getting tantalizingly close to 99% accuracy, leaving ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) in the dust and securing himself the first breakpoint of the Bracket Reset.

With a 5-6 lead, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) doubled down and picked HR1; a pick which he, too, previously lost against ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357).
By adding insult to injury on the home stretch, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) finished with an FC, replicating his opponent's feat from the previous match with much better accuracy to boot and denying his chance at another tiebreaker.

In the end, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073), the tournament favorite since the very beginning won, but absolutely no one could've anticipated what it took him to get there in the end. 

The Grand Finals of EGTS 2025, much like the entire tournament itself, was a spectacle second to none.

> To have two of the game mode's most prolific players push each other to a tiebreaker nearly twice is a historic moment that none of the hundreds of people watching live are sure to forget any time soon.

Reviewer 2: overhauled since

There are no words for what ended up happening.
 
<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-peaceful-vs-kanten_07-3.mp4" type="video/mp4" preload="none">
    </video>
</div>

::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) wins with one of the most decisive ways a tiebreaker could be, with a 3,775 points difference, and advances to the Semifinals in the Winner's Bracket that way.

### Semifinals: ::{ flag=KR }:: Peaceful vs. ::{ flag=JP }:: _Kan2

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/f4r-YGyv7k4" frameborder="0" allowfullscreen></iframe>
</div>

> We're now in the first round of Semifinals for the Loser's Bracket, with a legendary lineup of players to say the least, with ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) on one side, being known as one of the best tournament players in the game, with 4 different TWC podium finishes, and three of them being 2nd place in 2020, 2021, and 2022 all in a row, and being the current [Project Prism champion](https://osu.ppy.sh/community/forums/topics/1909279). On the other side, we have ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196), having one of the highest skillcaps for a tournament player, despite not playing in tournaments, showcased with his 2x miss on [TremENDouS](https://osu.ppy.sh/beatmapsets/1632807#taiko/3332914), EGTS 2021's Grand Finals Tiebreaker, being unmatched to this day, and being the only player that has won both Squad Global Taiko Showdowns to ever happen, in 2021, and 2023.

Reviewer 3: also confused on how it says `despite not playing in tournaments`, but they won both Squad Global Taiko Showdowns? what was meant here?
```suggestion
We're now in the first round of Semifinals for the loser's bracket, with a legendary line-up of players to say the least, with ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) on one side, being known as one of the best tournament players in the game, with 4 different TWC podium finishes, and being the current [Project Prism champion](https://osu.ppy.sh/community/forums/topics/1909279). On the other side, we have ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196), having one of the highest skill caps for a tournament player, despite not playing in tournaments, showcased with his 2x miss on [TremENDouS](https://osu.ppy.sh/beatmapsets/1632807#taiko/3332914), EGTS 2021's Grand Finals Tiebreaker, being unmatched to this day, and being the only player that has won both Squad Global Taiko Showdowns to ever happen, in 2021, and 2023.
```

> Now, how could this match between such juggernauts of osu!taiko go? Picks for both players were pretty unsurprising, with ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) choosing the reading aspect of the mappool, with [FM3](https://osu.ppy.sh/beatmapsets/2336583#taiko/5016186), [HR2](https://osu.ppy.sh/beatmapsets/2336561#taiko/5016142), [HD1](https://osu.ppy.sh/beatmapsets/2336375#taiko/5015685), [FM1](https://osu.ppy.sh/beatmapsets/2336556#taiko/5016129), [NM4](https://osu.ppy.sh/beatmapsets/2336545#taiko/5016102), and [NM6](https://osu.ppy.sh/beatmapsets/2336555#taiko/5016128), as a mechanics-oriented final pick. For ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196), we ended up with [NM1](https://osu.ppy.sh/beatmapsets/2336548#taiko/5016113), [NM3](https://osu.ppy.sh/beatmapsets/2336550#taiko/5016115), [NM5](https://osu.ppy.sh/beatmapsets/2336559#taiko/5016136), [HR1](https://osu.ppy.sh/beatmapsets/2336376#taiko/5015688), [NM2](https://osu.ppy.sh/beatmapsets/2336557#taiko/5016130), and [DT2](https://osu.ppy.sh/beatmapsets/2336377#taiko/5015689) all being picked. This entire match went as according for the first 4 picks, but ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) then lost on HD1, losing the slow Hidden reading for the second match in a row, and giving ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196) the first break point in this match. Sadly for him though, this would not be the last break point in the match, as ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) ended up retaliating, by first winning [HR1](https://osu.ppy.sh/beatmapsets/2336376#taiko/5015688) in dominating fashion by FC'ing it, and getting to 4-4, and finally, winning [DT2](https://osu.ppy.sh/beatmapsets/2336377#taiko/5015689) in one of the most impressive break points to happen, with a 483 points difference, to win the whole match.

Reviewer 3: this paragraph has the same issue as the other one, you are diluting the main point with just spamming all of their picks rather than writing what the skill that they weighed into was

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-peaceful-vs-_kan2.mp4" type="video/mp4" preload="none">
    </video>
</div>

### Grand Finals: ::{ flag=JP }:: Maimaing vs. ::{ flag=JP }:: kanten_07

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/NWrW5WXmIdE" frameborder="0" allowfullscreen></iframe>
</div>

Here we were, the final match deciding who would face off against ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073), in the Grand Finals of EGTS 2025 the following day.

> With everyone knowing both players were really good, everyone was expecting this match to be extremely close as it was, especially as it was a revenge match from Semifinals, where ::{ flag=JP }:: [Maimaing](https://osu.ppy.sh/users/14520910) came out on top, winning 7-5. 

Reviewer 3: ```suggestion
With everyone knowing both players were really good, everyone was expecting this match to be extremely close as it was, especially as it was a revenge match from Semifinals, where ::{ flag=JP }:: [Maimaing](https://osu.ppy.sh/users/14520910) came out on top, winning 7–5.
```

> However, this time around, despite the very similar scoreline, with a 6-7 for ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357), the story of the match was very different, with both players truly showing they were ready, and both players ended up winning their 6 respective picks, which is an absurd sight to have.

Reviewer 3: ```suggestion
However, this time around, despite the very similar scoreline, and with a 6–7 for ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357), the story of the match was very different, with both players truly showing they were ready, and both players ended up winning their 6 respective picks, which is an absurd sight to behold.
```

![](/wiki/shared/news/2025-05-03-expert-global-taiko-showdown-2025-concludes/maimaing-vs-kanten_07-1.png)

> With no choice but to play one of the longest and brutal tiebreakers in osu!taiko history to get a chance to continue playing in this tournament, both players would be starting the final map in this match.

Reviewer 3: ```suggestion
With no choice but to play one of the longest and brutal tiebreakers in osu!taiko history to get a chance to continue playing in this tournament, both players started the final map in this match.
```

![](/wiki/shared/news/2025-05-03-expert-global-taiko-showdown-2025-concludes/maimaing-vs-kanten_07-2.png)

> However, with the first quarter of the map done, it started looking hopeless, as ::{ flag=JP }:: [Maimaing](https://osu.ppy.sh/users/14520910) was already 23,145 points ahead, with ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) being 31 misses behind, this seemed like a done deal, especially with none of the song's hard parts having come up yet.

Reviewer 3: ```suggestion
However, with the first quarter of the map done, it started looking hopeless, as ::{ flag=JP }:: [Maimaing](https://osu.ppy.sh/users/14520910) was already 23,145 points ahead. With ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) being 31 misses behind, this seemed like a done deal, especially with none of the song's hard parts having come up yet.
```

And yet....

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-maimaing-vs-kanten_07.mp4" type="video/mp4" preload="none">
    </video>
</div>

![](/wiki/shared/news/2025-05-03-expert-global-taiko-showdown-2025-concludes/maimaing-vs-kanten_07-3.png)

> Whatever it was that happened in his mind to hit all of this, this was the turning point of the map, as ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) took the lead from that point, and never looked back, to win with a jaw-dropping 7,627 points difference. This whole Grand Finals is truly a once-in-a-lifetime experience to enjoy, so make sure to check both matches out.

Reviewer 3: ```suggestion
Whatever it was that happened in his mind to hit all of this, this was the turning point of the map, as ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) took the lead from that point, and never looked back, winning with a jaw-dropping 7,627 points difference. This whole Grand Finals is truly a once-in-a-lifetime experience to enjoy, so make sure to check both matches out.
```

> ## Original songs 

Reviewer 3: ```suggestion
## Original songs
```
